controllers/admin/AdminTrackingController.php

```diff
@@ -383,6 +385,29 @@ public function displayEnableLink($token, $id, $value, $active, $id_category = n
         $this->_helper_list->identifier = $this->identifier;
         $this->_helper_list->table = $this->table;
 
+        // Since Categories controller is migrated to Symfony
+        // it makes use of new endpoint instead of relaying on legacy controller
+        if ($this->list_id === 'empty_categories') {
```

Comment:
`'empty_categories'` must be a const

Reply:
I don't think it's worth introducing legacy const. There are 7 lists in this controller, each have their own id without constants. In addition, this controller is already migrated in `develop` branch.

Reply:
Ooohhhh, you're right, didn't notice the legacy controller ^^